### PR TITLE
foot: update to 1.9.2.

### PR DESCRIPTION
**Pull request by ifreund**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/foot/template [hunk 1]
```diff
@@ -25,6 +25,6 @@ post_install() {
 foot-terminfo_package() {
 	short_desc+=" - terminfo data"
 	pkg_install() {
-		vmove usr/share/terminfo
+		vmove usr/share/foot/terminfo
```
[the remark this answers is not in the document]
Comment (ram02z) [reply srcpkgs/foot/template:28]: The non standard terminfo location has a bunch of outstanding issues. See https://codeberg.org/dnkl/foot/issues/695

[the remark this answers is not in the document]
Comment (ifreund) [reply srcpkgs/foot/template:28]: Yep, from that issue and dnkl's mail to maintainers it looks like upstream now recommends exactly what @ericonr suggested.

**Comment by ram02z** [comment]
@ericonr sorry for the ping, but can this get merged?

**Comment by cinerea0** [comment]
Agreed, this would be good to get merged.

**Comment by ericonr** [comment]
Seems to be working here locally, but I was under the impression a packaging fix release would be coming? 1.9.1 or whatever?

**Comment by stacyharper** [comment]
For additional info, here a thread with @dnkl, the developer behind
foot:

https://gitlab.alpinelinux.org/alpine/aports/-/merge_requests/24357#note_177559

The actual mr that get merged into aports:

https://gitlab.alpinelinux.org/alpine/aports/-/merge_requests/24845

**Comment by cinerea0** [comment]
A packaging fix may be coming, but it doesn't look like it's coming soon.

**Comment by dnkl** [comment]
I can do a 1.9.1 release, if that makes it easier for you. I haven't yet, to avoid unnecessary packaging churn.

**Comment by cinerea0** [comment]
@ifreund It looks like 1.9.1 has just been released.

**Pull request merged**